### ISSUE TITLE
[1.7.10] [Request] Prevent Viewing Locked Recipes in NEI

### ISSUE THREAD

**Issue by DarianLStephens** [chylex/Hardcore-Ender-Expansion#208]
I would love an option to revert the behavior introduced in Release 1.8.4, where it no longer prevents viewing the recipes of HEE items with NEI's interface.
Personally, I feel it reduces the usefulness of the Compendium, since you can now just view anything's recipe without consequence. Perhaps just knowing the recipe doesn't help you that much, but besides restoring some of that feel, it would still help with usability, allowing one to just use NEI as normal and the Compendium will be opened as needed.
Basically, I would like a config option to restore the intercepting of NEI recipe keys for HEE items
If you're willing to go a little farther, you could possibly add a new function of preventing NEI from displaying recipes of items you have not unlocked in the Compendium, maybe even hiding them from NEI's display altogether until they're discovered/unlocked.

**Comment by chylex** [comment]
HEE 1 is no longer in development. You can disable NEI integration in the config using the S:IMC entry http://hee-api.chylex.com/#imc-system

**Comment by chylex** [comment]
Oh, I misread the request. You can press P in NEI to view HEE items in the compendium, but if you want the old feature back, you would have to build a custom version of the mod, sorry.

**Comment by DarianLStephens** [comment]
I'd be happy to set up a dev environment and build it, but I'm not so sure how comfortable I'd be altering code... You wouldn't happen to have a revision of whatever file was changed in that update, do you? I could try and work it out from there.

**Comment by chylex** [comment]
You can view the history of https://github.com/chylex/Hardcore-Ender-Expansion/blob/HEE1-1.7.10/src/main/java/chylex/hee/system/integration/handlers/NotEnoughItemsIntegration.java but it might be a bit of a mess because I replaced the repo at one point, so it might be tricky to find which version you're looking for. Going by the date of release is a good start.

**Comment by chylex** [comment]
This might be it, there were some versions of the file that crashed but it seems to be the latest version with that change - https://github.com/chylex/Hardcore-Ender-Expansion/blob/184bcab66bf86f3ef1f3303a2e4c9acdff64de96/src/main/java/chylex/hee/system/integration/handlers/NotEnoughItemsIntegration.java

Make sure you build the HEE1-1.7.10 branch, the files don't exist in master.

**Comment by DarianLStephens** [comment]
Thanks, I'll take a look at it tomorrow. Hopefully I can get something to work without destroying my computer!
